api/routes/web.php

```diff
@@ -15,3 +16,6 @@
     ->middleware(EnforceFeatureFlags::in([
         Features\SocialAuthentication::NAME,
     ]));
+
+// Backwards compatibility with Airlock
+Route::get('/airlock/csrf-cookie', [CsrfCookieController::class, 'show']);
```

Comment:
Should `airlock/csrf-cookie` not be `/sanctum/csrf-cookie` ?

Reply:
This is to preserve backwards compatibility with old versions of the frontend that will still call `/airlock/csrf-cookie`.

The `sanctum` route is automatically registered by Sanctum